demo/agilebank/dryrun/bad_resource/duplicate_ing.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
looks like other examples use `extensions/v1beta1` but this one has `networking.k8s.io/v1beta1` api version (which won't work on older clusters)

Reply:
good catch! done.